src/mavsdk/core/CMakeLists.txt

```diff
@@ -102,6 +107,7 @@ target_include_directories(mavsdk
     PUBLIC
     $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/include/mavsdk>
     $<BUILD_INTERFACE:${CMAKE_CURRENT_BINARY_DIR}/include/mavsdk>
+    $<BUILD_INTERFACE:${MAVLINK_INCLUDE_DIR}>
```

Comment:
Is that required? It should come with MAVLink::mavlink, right?

Reply:
I just tried without and it seems to be required for other targets like integration_tests, etc.

Reply:
:thinking:

Reply:
Because MAVLink is now a private dependency but the integration tests still require the headers.